app/src/main.ts

```diff
@@ -179,83 +179,107 @@ if (shouldQuit) {
     }
   });
 
-  app.on('ready', () => {
-    mainWindow = createMainWindow(appArgs, app.quit.bind(this), setDockBadge);
-    createTrayIcon(appArgs, mainWindow);
-
-    // Register global shortcuts
-    if (appArgs.globalShortcuts) {
-      appArgs.globalShortcuts.forEach((shortcut) => {
-        globalShortcut.register(shortcut.key, () => {
-          shortcut.inputEvents.forEach((inputEvent) => {
-            mainWindow.webContents.sendInputEvent(inputEvent);
-          });
+  if (appArgs.widevine !== undefined) {
+    // @ts-ignore
+    app.on('widevine-ready', (version, lastVersion) => {
+      console.log('widevine-ready', version, lastVersion);
+      ready();
+    });
+
+    // @ts-ignore
+    app.on('widevine-update-pending', (currentVersion, pendingVersion) => {
+      console.log(
+        `Widevine ${currentVersion} is ready to be upgraded to ${pendingVersion}`,
+      );
+    });
+
+    // @ts-ignore
+    app.on('widevine-error', (error) => {
+      console.error('WIDEVINE ERROR: ', error);
+    });
+  } else {
+    app.on('ready', () => {
+      console.log('ready');
+      ready();
+    });
+  }
+}
+
+function ready(): void {
```

Comment:
```suggestion
function onReady(): void {
```

for consistency with our other similar event handlers?